framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -67,6 +67,13 @@ void MediaPlayerImpl::createPlayer(player_result_t &ret)
 		return;
 	}
 
+	if (init_audio_stream_out() != AUDIO_MANAGER_SUCCESS) {
+		meddbg("MediaPlayer create fail : init_audio_stream_out\n");
```

Comment:
This debug should display return value of init_audio_stream_in to check error properly.

Reply:
Modified.